kwonhayeon/likelion-assignment-crud/src/main/java/com/example/likelionassignmentcrud/feedback/api/FeedbackController.java

```diff
@@ -0,0 +1,38 @@
+package com.example.likelionassignmentcrud.feedback.api;
+
+import com.example.likelionassignmentcrud.feedback.api.dto.request.FeedbackSaveRequestDto;
+import com.example.likelionassignmentcrud.feedback.api.dto.response.FeedbackListResponseDto;
+import com.example.likelionassignmentcrud.feedback.api.application.FeedbackService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/feedbacks")
+public class FeedbackController {
+
+    private final FeedbackService feedbackService;
+
+    // 피드백 저장
+    @PostMapping
+    public ResponseEntity<String> feedbackSave(@RequestBody FeedbackSaveRequestDto dto) {
+        feedbackService.feedbackSave(dto);
+        return new ResponseEntity<>("피드백 저장 완료!", HttpStatus.CREATED);
```

Comment:
공통 응답 형식을 만들어 사용하면 확장성과 통일성 부분에서 더욱 편리하게 관리할 수 있어요!

```suggestion
        return ResponseEntity.status(HttpStatus.CREATED)
        .body(new ApiResponse("피드백 저장 완료!", 201));
```